docs/ref/settings.txt

```diff
@@ -1260,6 +1260,31 @@ Default: ``'``:class:`django.db.models.AutoField`\ ``'``
 Default primary key field type to use for models that don't have a field with
 :attr:`primary_key=True <django.db.models.Field.primary_key>`.
 
+.. admonition:: Migrating auto-created through tables.
+
+    The value of ``DEFAULT_AUTO_FIELD`` will be respected when creating new
+    auto-created through tables for many-to-many relationships.
+
+    The primary key of auto-created through tables cannot currently be updated
+    by the migrations framework, however.
```

Comment:
Maybe:
```suggestion
    The value of ``DEFAULT_AUTO_FIELD`` will be respected when creating new
    auto-created through tables for many-to-many relationships. However, the
    primary keys of existing auto-created through tables cannot currently be
    updated by the migrations framework
```
or even `Unfortunately` instead of `However` :thinking:

Reply:
Let's go with Unfortunately, but I'll push it as a new paragraph for visual emphasis. (We can collapse if you don't like it.)